src/libs/mysqlHandler.go

```diff
@@ -1452,23 +1358,23 @@ func GetCiliumLogsMySQL(cfg types.ConfigDB, filterLog types.CiliumLog) ([]types.
 	return resLog, resTotal, err
 }
 
+// UpdateCiliumLogsMySQL -- Update existing log with time and count
 func UpdateOrInsertCiliumLogsMySQL(cfg types.ConfigDB, ciliumlogs []types.CiliumLog) error {
 	var err error = nil
 	db := connectMySQL(cfg)
 	defer db.Close()
 
-	for _, kubearmorLog := range ciliumlogs {
-		if err = updateOrInsertCiliumLogMySQL(db, kubearmorLog); err != nil {
+	for _, ciliumLog := range ciliumlogs {
+		if err := updateOrInsertCiliumLogMySQL(db, ciliumLog); err != nil {
 			log.Error().Msg(err.Error())
 		}
 	}
 	return err
 }
 
-// UpdateCiliumLogsMySQL -- Update existing log with time and count
 func updateOrInsertCiliumLogMySQL(db *sql.DB, ciliumlog types.CiliumLog) error {
```

Comment:
these two function names are a bit confusing, can we update these?

Reply:
Can you elaborate more. Why is this confusing ?

Reply:
one function is `updateOrInsertCiliumLogMySQL` and then above at line 1362 we have `UpdateOrInsertCiliumLogsMySQL()` both have similar names.